buildenv/jenkins/jck_sync

```diff
@@ -0,0 +1,106 @@
+        sh "rm -rf ${env.WORKSPACE}/aqa-tests/TKG"
+        // call cleanWs() again
+        cleanWs disableDeferredWipeout: true, deleteDirs: true
+    }
+}
+
+def syncJckMaterial(){
+        echo "Starting script to sync JCK materials ....."
+        def JCK_VERSION = params.JDK_VERSION ? "-j ${params.JDK_VERSION}": ""
+        def JCK_GIT_BRANCH = params.JCK_GIT_BRANCH ? "-gb ${params.JCK_GIT_BRANCH}": ""
+        def ARTIFACTORY_URL = params.ARTIFACTORY_URL ? "-au ${params.ARTIFACTORY_URL}": ""
+        def JCK_GIT_REPO = params.JCK_GIT_REPO ? "-repo ${params.JCK_GIT_REPO}": ""
+        stage('Build') {
+                script {
+                        getJavaSDK()
+
+                        withCredentials([
+                            usernamePassword(credentialsId: "${params.ARTIFACTORY_CREDENTIALS}",
+						usernameVariable: 'ARTIFACTORY_USER', passwordVariable: 'ARTIFACTORY_CREDENTIALS'),
+                            string(credentialsId: "${params.GIT_CREDENTIALS}",variable: 'GIT_CREDENTIALS')
+                        ]) {
+                                def currentDirectory = sh(script: 'pwd', returnStdout: true).trim()
+                                dir("${currentDirectory}/aqa-tests/jck/jck-semiauto-updater") {
+                                    sshagent(credentials:["${params.USER_CREDENTIALS_ID}"], ignoreMissing: true){
+                                        def SCRIPT = "./jckupdater.sh ${JCK_VERSION} -at ${ARTIFACTORY_CREDENTIALS} ${JCK_GIT_REPO}  -gt ${GIT_CREDENTIALS}  ${ARTIFACTORY_URL}  ${JCK_GIT_BRANCH}"
+                                        echo "${SCRIPT}"
+                                        def EXITCODE = sh(script: "${SCRIPT}", returnStatus: true)
+                                        echo "EXITCODE= ${EXITCODE}"
+                                        if (EXITCODE == 2) {
+                                            echo "Script returned exit code 2, No new update available. Marking the job as SUCCESS. "
+                                        } else if (EXITCODE == 0) {
+                                            echo "Script returned exit code 0, new update is available and PR is created."
+                                        } else {
+                                            error("script failed with exit code ${EXITCODE}")
+                                        }
+                                    }
+                                }
+                        }
+                    }
+                }
+    }
+
+//Download JAVA SDK 
+def getJavaSDK(){
+
+        JDK_VERSION_OPTION = params.JDK_VERSION ? "-j ${params.JDK_VERSION}" : ""
+        JDK_IMPL_OPTION = params.JDK_IMPL ? "-i ${params.JDK_IMPL}" : ""
+        CUSTOMIZED_SDK_URL_OPTION = "-c ${params.CUSTOMIZED_SDK_URL}"
+
+        GET_SH_CMD = "./get.sh -s `pwd`/.. -p $PLATFORM -r ${SDK_RESOURCE} ${JDK_VERSION_OPTION} ${JDK_IMPL_OPTION} ${CUSTOMIZED_SDK_URL_OPTION} "
```

Comment:
Please only call `getBinaryOpenjdk` and `testJavaVersion` in get.sh.

Reply:
Removed call for `getFunctionalTestMaterial`.
Calling `getBinaryOpenjdk`, `testJavaVersion` and `getTestKitGen`. 
`getTestKitGen` will be called because of absence of TKG directory. Also we need to call while incorporating `jck_target_folder_check` script.